OneSignal.js

```diff
@@ -64,6 +64,83 @@ var OneSignal = {
     xhr.open(action, ONESIGNAL_HOST_URL + url);
     xhr.setRequestHeader('Content-Type', 'application/json;charset=UTF-8');
     xhr.send(JSON.stringify(inData));
+  }
+};
+
+var __OSMessageHelper = {
+  // Dictionary of callbacks by type
+  // Each element is an array that can have a list of callbacks.
+  callbackHash: {},
```

Comment:
let's have a dictionary `portHash` so that we can store the respective object types in the appropriate dictionary

Reply:
Fixed